### PR TITLE
[improve][doc] SEO for Administration

### PR DESCRIPTION
**Pull request by zuobiao-zhou**
This PR do SEO for Administration

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete` 

image
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by zuobiao-zhou** [comment]
PTAL @Anonymitaet TIA~

File: docs/administration-geo.md [hunk 1]
```diff
@@ -33,21 +34,17 @@ Applications can create producers and consumers in any of the clusters, even whe
...
 
-This section guides you through the steps to configure geo-replicated clusters.
-1. [Connect replication clusters](#connect-replication-clusters)
-2. [Grant permissions to properties](#grant-permissions-to-properties)
-3. [Enable geo-replication](#enable-geo-replication)
-4. [Use topics with geo-replication](#use-topics-with-geo-replication)
+To configure geo-replicated clusters, complete the following steps.
 
-### Connect replication clusters
+### 1. Connect replication clusters
```
Comment (Anonymitaet) [inline docs/administration-geo.md:47]: Why not use "Step 1"?

File: docs/administration-zk-bk.md [hunk 2]
```diff
@@ -268,7 +274,12 @@ In Pulsar, you can set *persistence policies* at the namespace level, which dete
 
 You can set persistence policies for BookKeeper at the [namespace](reference-terminology.md#namespace) level.
 
-#### Pulsar-admin
+````mdx-code-block
```
Comment (Anonymitaet) [inline docs/administration-zk-bk.md:277]: Double check: have you previewed line 277 - 388 to make everything go as expected?

Comment (zuobiao-zhou) [reply docs/administration-zk-bk.md:277]: image
image
image

**Comment by zuobiao-zhou** [comment]
Updated, PTAL @Anonymitaet

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**